char/abstract/lef_gen

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+mkdir -p GDS
+
+extract=0
+gui=0
+noexit=0
+
+while test $# -gt 0
+do
+    case "$1" in
+        -extract) extract=1
+            ;;
+        -gui) gui=1
+            ;;
+        -noexit) noexit=1
+            ;;
+        -debug) noexit=1; gui=1
+            ;;
+    esac
+    shift    
+done
+
+cp ../../lib/12T/gds/*.gds GDS/
```

Comment:
can  you add a comment about what is being done here?

could the script be generalize to work with other variant than the `12T` one?

Reply:
probably...this is for LEF generation.  Don't think have this ready.  Can we wait?